src/content/docs/it/tutorial/1-setup/2.mdx

```diff
@@ -0,0 +1,147 @@
+</PreCheck>
+
+## Esegui l'installazione guidata di Astro
+
+Il metodo preferito per creare un nuovo sito Astro è tramite la nostra installazione guidata `create astro`.
+
+1. Nel promt dei comandi del tuo terminale, esegui il comando seguente usando il tuo package manager preferito:
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      # crea un nuovo progetto con npm
+      npm create astro@latest
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      # crea un nuovo progetto con pnpm
+      pnpm create astro@latest
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      # crea un nuovo progetto con yarn
+      yarn create astro
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+
+2. Conferma `y` per installare `create-astro`
+3. Quando il promt chiede, "Where would you like to create your new project?" inserisci il nome della cartella per creare una nuova directory per il tuo progetto, per esempio `./tutorial`
+
+    :::note
+    Un nuovo progetto Astro può solo essere creato in una cartella completamente vuota, quindi scegli un nome per la tua cartella che non esista già!
+    :::
+
+4. Vedrai una breve lista di template con cui iniziare. Usa le freccie (sopra e sotto) per navigare al template "Empty", e poi premi invio (enter) per inviare la tua scelta.
+
+5. Quando il promt chiede, "Would you like to install dependencies?" inserisci `y`.
+
+6. Quando il promt chiede se pianifichi di utilizzare TypeScript, inserisci `n`.
+
+7. Quando il promt chiede, "Would you like to initialize a new git repository?" inserisci `y`.
+
+Quando hai completato l'installazione guidata, non hai più bisogno di questo terminale. Ora puoi aprire VS Code per continuare.
+
+## Apri il tuo progetto in VS Code
+
+1. Apri VS Code. Ti verrà richiesto di aprire una cartella. Seleziona la cartella che hai creato durante l'installazione guidata.
+
+2. Se questa è la prima volta che apri un progetto Astro, dovresti vedere una notifica che ti chiede se vuoi installare le estensioni consigliate. Clicca per vedere le estensioni consigliate, e scegli la "[Astro language support extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode)". Questa fornirà evidenziazione della sintassi e completamento automatico del codice Astro.
+
+3. Assicurati che il terminale sia visibile e che puoi vedere il promt dei comandi, per esempio:
+
+    ```sh
+    user@machine:~/tutorial$
+    ```
+
+    :::tip[Scorciatoia da tastiera]
+    Per alternare la visibilità del terminal, usa <kbd>Ctrl + J</kbd> (macOS: <kbd>Cmd ⌘ + J</kbd>).
+    :::
+
+Ora puoi utilizzare il terminale circoscritto in questa finestra, invece del terminale del tuo computer, per il resto di questo tutorial.
+
+
+
+## Esegui Astro in modalità sviluppo (dev)
+
+In modo di avere una preview dei file del tuo progetto _come un sito_ mentre lavori, avrai bisogno che Astro sia eseguito in modalità sviluppo (dev).
```

Comment:
Eviterei il termine inglese, si può evitare qui... `preview` viene usato in italiano, ma esiste una traduzione.

```suggestion
Per avere un'anteprima dei file del tuo progetto _come un sito_ mentre lavori, avrai bisogno che Astro sia eseguito in modalità sviluppo (dev).
```